systest/backup/multi-tenancy/backup_test.go

```diff
@@ -60,7 +60,7 @@ func TestBackupMultiTenancy(t *testing.T) {
 	galaxyToken := testutil.Login(t, galaxyCreds)
 
 	// Create a new namespace
-	ns, err := testutil.CreateNamespace(t, galaxyToken)
+	ns, err := testutil.CreateNamespaceWithRetry(t, galaxyToken)
```

Comment:
We don't need `CreateNamespaceWithRetry` now.

Reply:
we actually do. even if fails after retry we don't want tests to fail. this will retry to get a successful response